lang/en_US.json

```diff
@@ -1,16 +1,313 @@
+		"FORUM__PHRASE__HISTORY__EDIT__MSG": "has just modified a message:",
+		"FORUM__PHRASE__HISTORY__REPORT__MSG": "just posted a message:",
+		"FORUM__PHRASE__HISTORY_FOR__REASON": "for the following reason:",
+		"FORUM__PHRASE__HISTORY__EDITPERM__USER": "changed the permissions of:",
+		"FORUM__PHRASE__HISTORY__CREATE__FORUM": "just created a new forum:",
+		"FORUM__PHRASE__FLASH__NOTREPORT__MSG": "You can not carry forward your own messages",
+		"FORUM__PHRASE__HISTORY__EDIT__CATEGORY": "Category successfully edited",
+		"FORUM__PHRASE__HISTORY__DELETE__TOPIC": "just deleted a topic",
+		"FORUM__PHRASE__HISTORY__DELETE__MSG": "just deleted a topic",
+		"FORUM__PHRASE__HISTORY__STICK__TOPIC": "just put forward a topic",
+		"FORUM__PHRASE__HISTORY__UNSTICK__TOPIC": "just added a topic",
+		"FORUM__PHRASE__HISTORY__LOCK__TOPIC": "just locked a topic",
+		"FORUM__PHRASE__HISTORY__UNLOCK__TOPIC": "just unlock a topic",
+		"FORUM__PHRASE__HISTORY__EDIT__GROUP": "just edited a group",
+		"FORUM__PHRASE__PAGE__BANNED": "You are banished!",
+		"FORUM__PHRASE__PAGE__BANNED_1": "You have been banned until",
+		"FORUM__PHRASE__PAGE__BANNED_2": "by",
+		"FORUM__PHRASE__PAGE__BANNED_3": "for the following reason:",
+		"FORUM__PHRASE__PAGE__WORD_1": "Here you can decide which words are forbidden, as well as the expression by which the word must be replaced.",
+		"FORUM__PHRASE__PAGE__WORD_2": "This tool is a first defense of the moderation of your forum.This tool allows you to automatically replace certain words as soon as there is a new message or a message edition.",
+		"FORUM__PHRASE__PAGE__WORD_3": "This tool is not case-sensitive, so forum, Forum, FORUM, FoRuM will be replaced by what you have indicated.",
+		"FORUM__PHRASE__PAGE__REPORT_1": "Moderators can access this section from this page:",
+		"FORUM__PHRASE__PAGE__PERM_1": "This tool is very useful when you add new ranks and you want to give them permissions, by default access is restricted if no permissions are set for the group.",
+		"FORUM__PHRASE__ONLINE": "online",
+		"FORUM__PHRASE__PAGE__ADMININDEX_1": "You have an idea of ​​improvement? Found, a bug? Or you want new features?",
+		"FORUM__PHRASE__PAGE__ADMININDEX_2": "Thank you for following this",
+		"FORUM__PHRASE__PAGE__ADMININDEX_3": ", you will find a questionnaire that you can fill if you want to improve the plugin and participate in its development.",
+		"FORUM__PHRASE__PAGE__ADMININDEX_4": "The forum at a rate of update very sustained, a message will appear above if there is a more recent version of the plugin.",
+		"FORUM__PHRASE__PAGE__ADMININDEX_5": "Warning, these buttons will delete all posts, topics, forums!",
+		"FORUM__PHRASE__PAGE__ADMININDEX_6": "Attention, pressing this button will delete all the history of the actions of the forum.",
+		"FORUM__PHRASE__PAGE__ADMININDEX_7": "Below, you can find the list of your suggestions.These suggestions will be added in later versions of the Forum plugin.",
```

Comment:
Theses* suggestions
I'm not sure for later versions, maybe futures or nexts versions ?